apps/fz_http/lib/fz_http_web/channels/gateway_channel.ex

```diff
@@ -9,33 +9,24 @@ defmodule FzHttpWeb.GatewayChannel do
   require Logger
 
   @impl Phoenix.Channel
-  def join("gateway:all", _payload, socket) do
-    {:ok, socket}
-  end
-
-  @impl Phoenix.Channel
-  def join("gateway:" <> id, payload, socket) do
```

Comment:
Are you sure that you want to change the topic to one for all gateways? We will have just one ID in the database so it will work like that anyways, but you will need to change it back as soon as we release 0.8 to support multisite...

Reply:
I rather keep it as simple as possible for now and we can think exactly how we handle multi-site when we start working on it.